db-developments/python/geocode_dob.py

```diff
@@ -18,20 +18,45 @@ def geocode(input):
 
     try:
         geo = g["1B"](
-            street_name=sname, house_number=hnum, borough=borough, mode="regular"
+            street_name=sname,
+            house_number=hnum,
+            borough=borough,
+            mode="regular",
         )
         geo = parse_output(geo)
-        geo.update(dict(uid=uid, mode="regular", func="1B", status="success"))
+        geo.update(
+            dict(
```

Comment:
might be a nice time to clean these up and just use a dictionary literal, e.g. {"uid"=uid...}